_data/gremien.yml

```diff
@@ -45,39 +47,39 @@ vorstand:
   - name: "Öffentlichkeitsarbeit"
     email: "oeffentlichkeitsarbeit@pep-dortmund.org"
     people:
-      - name: "Mira Arndt"
-        image: "/images/vorstand/mira.jpg"
+      - name: "Kevin Schmitz"
+        image: "/images/vorstand/kevinS.jpg"
 
   - name: "Alumniarbeit"
     email: "alumniarbeit@pep-dortmund.org"
     people:
-      - name: "Marie Schmitz"
-        image: "/images/vorstand/marie.jpg"
+      - name: "Karl Schiller"
+        image: "/images/vorstand/karl.jpg"
 
   - name: "Sommerakademie"
     email: "sommerakademie@pep-dortmund.org"
     people:
-      - name: "Rene-Marcel Lehner"
+      - name: "Anita Kohlwes"
         image: "/images/vorstand/rene.jpg"
 
   - name: "Wirtschaft"
     email: "wirtschaft@pep-dortmund.org"
```

Comment:
this should be physikon at pep-dortmund dot org

Reply:
done